### DISCUSSION TITLE
How to add translation?

### DISCUSSION THREAD

**Discussion by kvvloten** [linagora/linshare#328] [Q&A]
Hi,

I would like to add a new language to the interfaces for user, admin and upload. What should I change where to get that done?

- Kees.

**Comment by wboudiche** [comment]
Hi,
**For front ends :**
UI User : https://github.com/linagora/linshare-ui-user/tree/master/app/i18n/original
UI Admin : https://github.com/linagora/linshare-ui-admin/tree/master/src/core/plugins/i18n/locales
Upload request : https://github.com/linagora/linshare-ui-upload-request/tree/master/src/i18n/locales

**For Backend :**
You need to translate Email templates in the database
